hack/local-up-cluster.sh

```diff
@@ -16,7 +16,11 @@
 
 # This command builds and runs a local kubernetes cluster. It's just like
 # local-up.sh, but this one launches the three separate binaries.
-# You may need to run this as root to allow kubelet to open docker's socket.
+# Heed the warning below if not running as root...
+if [ “$(id -u)” != “0” ]; then
+    echo “WARNING : This script MAY be run as root for docker socket / iptables functionality... if failures occur... Retry as root.” 2>&1
```

Comment:
The quotes on these two lines are smart quotes (left double quotation mark `U201C` and right double quotation mark `U201D`) and not normal quotes. Please change them.